Podfile

```diff
@@ -0,0 +1,7 @@
+target 'Mobile Center Sample' do
```

Comment:
You probably want the specify the min iOS version supported, which is 10.9 for MC SDK